tensorflow/tensorflow.bzl

```diff
@@ -2653,6 +2653,31 @@ def gpu_py_test(
 def cuda_py_test(*args, **kwargs):
     gpu_py_test(*args, **kwargs)
 
+def tensorrt_py_test(test_names, tags):
```

Comment:
Can we move this change to python/compiler/tensorrt/tensorrt.bzl like https://github.com/tensorflow/tensorflow/pull/56497?

Reply:
@bixia1: I think that we can do it, but only after the merge of [#56497](https://github.com/tensorflow/tensorflow/pull/56497) because `tensorflow/python/compiler/tensorrt/tensorrt.bzl` is a new file and it is not in [the master](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/compiler/tensorrt) yet.

Reply:
I meant, adding this file here. Also, Jonathan said he would talk to you about coordinating this change and [56497](https://github.com/tensorflow/tensorflow/pull/56497?) , as they are making similar simplification.

Reply:
DONE